java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -319,6 +322,17 @@ public CompletableFuture<Long> scard(String key) {
         return commandManager.submitNewCommand(SCard, new String[] {key}, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Long> clientId() {
+        return commandManager.submitNewCommand(ClientId, new String[0], this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> clientGetName() {
```

Comment:
Same for `clientGetName`.
`clientGetName` without a route is not available in the cluster client.